docs/GITHUB_ACTIONS.md

```diff
@@ -51,6 +51,19 @@ The project uses 5 GitHub Actions workflows with the following execution order a
    - **Safety:** Requires typing exact confirmation string for manual execution
    - **Dependencies:** None - can be run independently
 
+6. **Cleanup Old Summarizer Container Images** (`.github/workflows/cleanup-old-images.yml`)
+   - **Trigger:** **Manual only** - workflow_dispatch
+   - **Purpose:** Deletes old container images from Quay.io to manage storage
+   - **Actions:**
+     - Processes 3 container images: aiobs-metrics-ui, aiobs-metrics-alerting, aiobs-mcp-server
+     - Deletes tags older than retention period (default: 30 days)
+     - Protects `latest` tag and all `*-release` tags
```

Comment:
Regarding the release image tag:
what is the need of adding release `v1.0.0-release` . can we keep it simple `v1.0.0` WDYT ?

Reply:
We can have something like this then:
  - Build workflow creates: 1.0.0, latest
  - Create Release workflow creates: v1.0.0 (with `v` prefix)
  - Cleanup protects anything matching `v*` pattern

This will make it more standard (GitHub releases use v prefix)